bot/exts/moderation/defcon.py

```diff
@@ -196,7 +203,14 @@ async def unshutdown(self, ctx: Context) -> None:
         role = ctx.guild.default_role
         permissions = role.permissions
 
-        permissions.update(send_messages=True, add_reactions=True, connect=True)
+        permissions.update(
+            send_messages=True,
+            add_reactions=True,
+            create_private_threads=None,
+            create_public_threads=None,
```

Comment:
Following the discussion in https://github.com/python-discord/bot/pull/1815#discussion_r711755032 I think these two lines and the matching lines in `shutdown` should be removed. I don't think there even is a None option for global permissions.